src/main/java/com/ge/predix/log/filter/LogFilter.java

```diff
@@ -87,8 +87,11 @@ public LogFilter() {
     @Override
     protected void doFilterInternal(final HttpServletRequest request, final HttpServletResponse response,
             final FilterChain filterChain) throws ServletException, IOException {
-
-        String correlationId = UUID.randomUUID().toString();
+    	String correlationId = request.getHeader(LOG_CORRELATION_ID);
+        if (StringUtils.isEmpty(correlationId)) {
+                correlationId = UUID.randomUUID().toString();
```

Comment:
Formatting is off. Recommend importing the ACS project format and code cleanup configurations.